Update documentation in RBM website

YifanCheng opened the pull request:
Mainly two documentation get updated
1) Documentation/tutorial
2) Documentation/steps to run RBM
The link to the RBM website is: http://rbm.readthedocs.io/en/latest/

Commit:
update documentation (run_RBm and tutorial)

YifanCheng committed:
Merge pull request #12 from YifanCheng/VIC_RBM2.2

Update documentation and add badge in RBM github repo

arbennett commented on docs/Documentation/run_RBM.md:
```diff
@@ -1,66 +1,64 @@
 ## 3. Run the Model
 ### Step 1 Build the forcing function files (flow and meteorology)
-In the example problem, I used **VIC4.2.d** with the two global parameter files in the directory, **run_VIC**. To create the necessary files (executable file to run VIC), you need to download [VIC4.2.d](https://github.com/UW-Hydro/VIC/releases/tag/VIC.4.2.d). The instructions to generate the executable file **vicNl** and run VIC can be found [here](http://vic.readthedocs.io/en/vic.4.2.d/). Copy t ...
+In the example, we run hydrological model **VIC4.2.d** to generate flow and energy fluxes, which are input data for RBM. The exmaple global parameter files can be found in the directory **run_VIC**. To create the necessary files (executable file to run VIC), you need to download [VIC4.2.d](https://github.com/UW-Hydro/VIC/releases/tag/VIC.4.2.d). The instructions to generate the executable file ** ...
```
There's a typo here: `exmaple` -> `example`

arbennett commented on docs/Documentation/run_RBM.md:
```diff
@@ -1,66 +1,64 @@
 
 ### Step 2 Generate topology file
-In the folder, ***../Test_Case***, run the perl script, **build_network_beta.pl**, (copied from ***../Perl_Scripts***) using the direction file that was created as described in the VIC model development (see [VIC model](https://uw-hydro.github.io/code/)). All the grid cells surrounding the basin of interest must contain a negative one (-1) for purposes of determining the headwaters segments. The  ...
+In the folder, ***../Test_Case***, run the perl script, **build_network_beta.pl** (copied from ***../Perl_Scripts***) using the direction file. Example direction file **Salmon_flowd_0.5.dir** can be found in ***../Test_Case***. In the flow direction file, all the grid cells surrounding the basin of interest must contain a negative one (-1) for purposes of determining the headwaters segments. Also ...
```
Some wording on the second to last sentence.  Should read something like `Also, a simulated river system...` or `Also, simulated river systems...`

arbennett commented on docs/Documentation/run_RBM.md:
```diff
@@ -1,66 +1,64 @@
 ### Step 3 Prepare control file
 
-Prepare a control file (see Figure 1 for an example) that describes:
+Prepare a control file (see [Figure 2](../figures/tutorial_f2.png) for an example) that describes:
```
Could make this into a gist or just a text snippet for ease of use.

YifanCheng replied:
An example control file is given in the example dataset and a table below that sentence explains line by line in that file. It might not be very proper to show the example control file there.

arbennett commented on docs/Documentation/run_RBM.md:
```diff
@@ -89,7 +87,7 @@ This will create the executable,
 For this example, copy **rout** to the folder **Test_Case**
 
-Create control file, **salmon.inp_DA**, to run the modified Lohmann routing model using the file, **Rout.Cells.init**, the first time and the file, **Rout.Cells**, if the routing model, **rout** is run again for the same set of unit hydrographs. The input file (in this example, **salmon.inp_DA**) is similar to that described on the VIC model Web site with the exception of the addition the Leopold ...
+Create control file **salmon.inp_DA** to run the modified Lohmann routing model using the file **Rout.Cells.init** the first time or the file **Rout.Cells** if the routing model **rout** is run again for the same set of unit hydrographs. The input file (in this example, **salmon.inp_DA**) is similar to that described on the VIC model Web site with the exception of the addition the Leopold paramet ...
```
Might read better if it's `Create the control file`

arbennett replied:
Also, after `**Rout.Cells.init** have `(for the first time)` with the parenthesis for a bit more clarity.

arbennett commented on docs/Documentation/tutorial.md:
```diff
@@ -1,23 +1,25 @@
         - daily maximum and minimum temperature
         - wind speed
-    *  VIC implements algorithms which are able to generate a full set of meteorological variables (see model output section for a list of meteorological variables) at daily or sub- daily time steps. These variables can also be directly given as input.
+    *  VIC includes a meteorological data disaggregator that calculates subdaily meteorological variables required by VIC from the minimum forcing variables. Details can be seen in [VIC website](http://vic.readthedocs.io/en/develop/Documentation/Drivers/Classic/ForcingData/).
```
`seen in the [VIC website]`

arbennett reviewed (changes requested):
A general comment: It might be easier to edit/review if the line width were taken down a bit.

DarriEy commented:
Hi Yifan

Some thoughts on the Tutorial and Steps pages:

Tutorial: 

The overview, Input and Output sections are all clear and descriptive. Regarding the Salmon example it would be nice to have a little bit more detail like a short description of the example area but maybe the whole appendix A should be moved to the Steps page? 

Steps to run RBM:

The steps are descriptive and relatively easy to understand, links to all the supporting data are provided which is great. My suggestions

•	Move Appendix A to the steps page. It is only referenced there and not in the tutorial page
•	Some slight information about the example case (e.g. map of Salmon river). Doesn‘t have to be much, just to give some physical info on the example
•	It’s great to have a link to example files (both input and output) like you do in step 2. It would be good to add such links to subsequent steps also.
•	Flow chart of model interactions and hierarchy would be great to easily understand how the flow of data between the models are processed (Should maybe be on Tutorial page)
•	Naming of the page should maybe be something like “Example run of VIC-RBM model” as the tutorial and steps seem to be referring to similar things.
•	Step 6, typo, routing instead of rounting

YifanCheng commented:
@DarriEy Thanks for your comments!

- Fixed it!
- I would add that as an issue that we would do for VIC_RBM2.3
- Fixed it! I add an hyperlink to figure 3 in step 5 to `prepare for routing input file`. As for the network file, it is too long to fit a figure.
- I would also add that as an issue that we would work on for VIC_RBM2.3
- Good point! I prefer to change to `How to run RBM (Salmon River Example)`
- Fixed it!
